server.js

```diff
@@ -0,0 +1,100 @@
+const express = require("express");
+const ramenData = require("./ramenData")
+const path = require('path')
+
+
+const app = express();
+
+// middleware for css stylesheet
+const staticMiddleware = express.static(path.join(__dirname, 'public'))
+
+app.use(staticMiddleware)
+
+//logging middleware
+const morgan = require("morgan");
+
+app.use(morgan('dev'));
+
+//pulls ramenData
+const noodles = ramenData.list()
+
+app.get("/", (req, res, next) => {
+    const html = `
+    <html>
+        <head>
+            <title>Ramen Reviews</title>
+            <link rel='stylesheet' href='/styles.css' />
+        </head>
```

Comment:
see how you're using this exact code in other routes? you can pull it out to a separate variable like
```js
const head = `
  <head>
         <title>Ramen Reviews</title>
         <link rel='stylesheet' href='/styles.css' />  
  </head>
`;
```
then use it here like 
```html
<html>
  ${head}
  <body>
    ... the rest of the code
  </body>
</html>
```

to make sure you dont repeat code too often